opacus/privacy_engine.py

```diff
@@ -300,18 +305,21 @@ def make_private(
                 every parameter tensor. Flat clipping is usually preferred, but using
                 per layer clipping in combination with distributed training can provide
                 notable performance gains.
+            noise_generator: torch.Generator() object used as a source of randomness for
+                the noise
 
         Returns:
-            Tuple of (model, optimizer, data_loader)
+            Tuple of (model, optimizer, data_loader).
+
             Model is a wrapper around the original model that also computes per sample
```

Comment:
I am actually curious to see how this will render on the website. As far as I am concerned, under Returns, only one paragraph can be written (i.e. no new line).